pebblo/reports/html_to_pdf_generator/report_generator.py

```diff
@@ -63,7 +63,7 @@ def convert_html_to_pdf(data, output_path, template_name, search_path, renderer)
         findings_count = data["reportSummary"].get("findings", 0)
         clientVersion = ""
         versionObj = data.get("clientVersion", None)
-        if versionObj is not None and versionObj.get("version", None) is not None:
+        if versionObj not in [None, {}] and versionObj.get("version", None) is not None:
```

Comment:
We do not need to explicitly mention None here

Reply:
When `clientVersion` is not there in data, it returns as `None`, but just to avoid error or exception we should keep it as `not in [None, {}]`

Reply:
Updated